app/views/users/index.html.erb

```diff
@@ -1 +1,6 @@
-<h1> List of user name and the number of posts <h1>
\ No newline at end of file
+<%= stylesheet_link_tag('user_card') %>
+<section>
+    <% @users.each do |user| %>
+    <%= render "user_card", user: user %>
+    <%end%>
+</section>
```

Comment:
- [ ] Currently, on the home page there is nothing being displayed there due to the error that I didn't notice. This is how it looks 👇 
![image](https://user-images.githubusercontent.com/90222110/230494453-4e0f0366-ddef-4474-9d13-02f75aecd5c0.png)

I have added some seed data but I can see them. Due to that, I can't proceed to check the posts and comments, so please correct it 🙏